### DISCUSSION TITLE
Introduce Increase In `HTTP Payload size limit` Request

### DISCUSSION THREAD

**Discussion by AyushSehrawat** [deta/cloud-docs#374] [Help] [answered]
Hello! I was making a cdn with deta Base and Drive, which worked perfectly fine locally. But when i see it on the deployed url and try to upload a file or stream a file ( used a 14mb image, works fine with small images ) it gives error.

When trying to fetch it - 
![image](https://user-images.githubusercontent.com/69469790/184305972-ef683162-1bd2-44ac-a665-4f2250011172.png)

When trying to upload it - 
![image](https://user-images.githubusercontent.com/69469790/184306247-da30c159-f588-47d2-a82f-46b183b72381.png)

It would be better if there was some support allowed to increase http payload ( currently 5.5mb for all)

**Comment by AyushSehrawat** [comment]
Though there is no issue with the manual upload from the dashboard, it would be nice if there was some workaround for it

**Comment by abdelhai** [comment, marked as answer]
This is a technical limitation and can't be changed atm unfortunately

**Reply by AyushSehrawat** [reply]
Hm... Any workaround you know @abdelhai  which can be implemented in fastapi/starlette ?
